templates/workspaces/base/terraform/variables.tf

```diff
@@ -35,3 +35,9 @@ variable "app_service_plan_sku" {
   default     = "P1v3"
   description = "App Service Plan SKU"
 }
+
+variable "storage_is_exposed_externally" {
+  type        = bool
+  default     = false
```

Comment:
Could you wire this up to "enable_local_debugging" and have one place where you turn this on?